src/lib/functions/local-proxy.js

```diff
@@ -34,8 +36,11 @@ const runFunctionsProxy = ({ binaryPath, context, directory, event, name, timeou
     '--timeout',
     `${timeout}s`,
   ]
+  const proxyProcess = execa(functionsProxyPath, parameters)
+
+  proxyProcess.stderr.pipe(stdout)
```

Comment:
What about checking for `proxyProcess.stderr && proxyProcess.stderr.pipe` existence? This would remove the need to update the mock.

Reply:
Hmm. 🤔 

My instinct is to say that I would rather keep the runtime code simple and shift any complexity to the test suite, and in reality there's no reason to perform that check other than the fact that we sometimes mock `execa` in tests.

Having said that, I don't have a strong preference here, so I can change it if you like.

Reply:
> I would rather keep the runtime code simple and shift any complexity to the test suite,

Yes, that's a good point. :+1: